content/docs/capabilities/ppl.mdx

```diff
@@ -316,6 +319,73 @@ Users can [find their device IDs](/docs/capabilities/device-identity#find-the-de
 
 :::
 
+### Record Matcher
+
+The record matcher is an object that uses operators as keys. It points to an integrated [external data source](/docs/capabilities/integrations) file (known as a "record") defined in the Enterprise Console. Pomerium matches requests to a specific external data source using a record's [foreign key](/docs/capabilities/integrations#foreign-key). You can use data stored in a record as external context in an authorization policy.
```

Comment:
This:

> It points to an integrated [external data source](/docs/capabilities/integrations) file (known as a "record") defined in the Enterprise Console.

sounds confusing to me... We use the word "collected" in the other doc. Maybe

> It points to records collected from an integrated [external data source](/docs/capabilities/integrations) defined in the Enterprise Console.

Reply:
Thanks for the suggestion. I'll update with your wording.